js/classes/entities/foreground/hud.js

```diff
@@ -22,6 +23,37 @@ class Hud {
         return 10;
     }
 
+    swapToCrosshair() {
+        let crosshairCursorUnclicked = 'url(../sprites/crosshair_unpressed.png) 16 16, auto';
+        let crosshairCursorClicked = 'url(../sprites/crosshair_pressed.png) 16 16, auto';
+
+        document.body.style.cursor = crosshairCursorUnclicked;
+
+        document.body.addEventListener('mousedown', () => {
+            document.body.style.cursor = crosshairCursorClicked;
```

Comment:
Interesting, so you had to modify CSS of the HTML document to replace the cursor image? Nice work.

Reply:
Yup, and it worked out pretty well. Thanks!